.github/workflows/main.yml

```diff
@@ -6,14 +6,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go_version: ["1.18.1", "1.17.6", "1.16.5"]
+        go_version: ["1.20", "1.19", "1.18"]
     steps:
       - uses: actions/checkout@v3
       - name: Setup Go
         uses: actions/setup-go@v3.2.0
         with:
           go-version: ${{ matrix.go_version }}
-      - run: ./.ci.gogenerate.sh    
-      - run: ./.ci.gofmt.sh
+      - name: Go generate
+        if: ${{ matrix.go_version != 1.18 }}
```

Comment:
Why are we excluding `gogenerate.sh` and `gofmt.sh` from running if it's Go 1.18?

Reply:
It's because it failed. I wrote a comment about it here: https://github.com/stretchr/testify/pull/1379#discussion_r1189008882

If we don't want this extra logic, we could also drop support for 1.18, maybe it's old enough at this point?

Reply:
That's fair -> https://endoflife.date/go

We can drop 1.18 support.

Reply:
@boyan-soubachov Updated to drop 1.18!